src/pages/home/Home.js

```diff
@@ -0,0 +1,60 @@
+import React, {useEffect, useState} from 'react';
+import "./Home.css";
+import {Link, NavLink} from "react-router-dom";
+import axios from "axios";
+import Header from "../../components/header/Header";
+import Footer from "../../components/footer/Footer";
+import logo from "../../assets/logo.png";
+import ErrorMessage from "../../components/errorMessage/ErrorMessage";
+
+
+const Home = () => {
+    const [reddit, setReddit] = useState([])
```

Comment:
Als `reddit` uiteindelijk een object zal worden, dan zou ik de default waarde aanpassen naar `undefined`. Dan heb je een duidelijker verschil tussen of het al wél of niet is ingeladen
```suggestion
    const [reddit, setReddit] = useState();
```